hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/snapshot/InfoSnapshotHandler.java

```diff
@@ -56,7 +56,7 @@ protected void execute(OzoneClient client, OzoneAddress address)
         .getSnapshotInfo(volumeName, bucketName, snapshotName);
 
     if (isVerbose()) {
-      err().printf("Snapshot info for snapshot: %d under o3://%s/ %s ",
+      err().printf("Snapshot info for snapshot: %s under o3://%s/%s %n",
```

Comment:
Is there a need of ` %n`?
```suggestion
      err().printf("Snapshot info for snapshot: %s under o3://%s/%s",
```

Reply:
This is for better representation of the output. Without %n the opening bracket starts in the same line after o3 url.

Reply:
Shouldn't it be `\n` and not `%n`?